site/content/docs/5.0/components/stepped-process.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+title: Stepped process
+description: Stepped process bar used for multiple steps forms process
+group: components
+toc: true
+---
+
+## How to use
+
+Use a `nav` element with `.stepped-process` class, containing an ordered list `<ol>`  with `.stepped-process-item` class on list items.
+
+Be careful to choose short labels when using multiple steps process, otherwise label will be cut to preserve inline layout. More information can be carried on the `title` attribute of the `.stepped-process-link`.
+
+Use `.current` class to define current step, alongwith with `aria-current="step"`.
```

Comment:
```suggestion
Add `.active` to a `.stepped-process-item` to indicate the current step, alongside `aria-current="step"` to convey the active state to assistive technologies.
```

BTW, we should move from `.current` to `.active` because `.active` is used throughout Bootstrap component as a convention, whereas `.current` only pops out in our custom components…